ansible/configs/ans-tower-prod/env_vars.yml

```diff
@@ -0,0 +1,380 @@
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+      - name: BasTowerTcp
+        description: "ALL from bastion tcp"
+        from_port: 0
+        to_port: 65535
+        protocol: tcp
+        group: BastionSG
+        rule_type: Ingress
+      - name: BasTowerUdp
+        description: "ALL from bastion udp"
+        from_port: 0
+        to_port: 65535
+        protocol: udp
+        group: BastionSG
+        rule_type: Ingress
+      - name: AllInternaltcp
+        description: "All other nodes tcp"
+        from_port: 0
+        to_port: 65535
+        protocol: tcp
+        group: HostSG
+        rule_type: Ingress
+      - name: AllInternaludp
+        description: "All other nodes udp"
+        from_port: 0
+        to_port: 65535
+        protocol: udp
+        group: HostSG
+        rule_type: Ingress
+      - name: AllTowerNodestcp
+        description: "All tower nodes tcp"
+        from_port: 0
+        to_port: 65535
+        protocol: tcp
+        group: TowerSG
+        rule_type: Ingress
+      - name: AllTowerNodesudp
+        description: "All tower nodes udp"
+        from_port: 0
+        to_port: 65535
+        protocol: udp
+        group: TowerSG
+        rule_type: Ingress
+  - name: HostSG
+    rules:
+      - name: HostUDPPorts
+        description: "Only from Itself udp"
+        from_port: 0
+        to_port: 65535
+        protocol: udp
+        group: HostSG
+        rule_type: Ingress
+      - name: Postgresql
+        description: "PostgreSql"
+        from_port: 5432
+        to_port: 5432
+        protocol: tcp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+      - name: HostTCPPorts
+        description: "Only from Itself tcp"
+        from_port: 0
+        to_port: 65535
+        protocol: tcp
+        group: HostSG
+        rule_type: Ingress
+      - name: TowerUDPPorts
+        description: "Only from tower"
+        from_port: 0
+        to_port: 65535
+        protocol: udp
+        group: TowerSG
+        rule_type: Ingress
+      - name: TowerTCPPorts
+        description: "Only from tower"
+        from_port: 0
+        to_port: 65535
+        protocol: tcp
+        group: TowerSG
+        rule_type: Ingress
+      - name: BastionUDPPorts
+        description: "Only from bastion"
+        from_port: 0
+        to_port: 65535
+        protocol: udp
+        group: BastionSG
+        rule_type: Ingress
+      - name: BastionTCPPorts
+        description: "Only from bastion"
+        from_port: 0
+        to_port: 65535
+        protocol: tcp
+        group: BastionSG
+        rule_type: Ingress
+
+instances:
+  - name: "bastion"
+    count: 1
+    unique: true
+    public_dns: true
+    dns_loadbalancer: true
+    security_groups:
+      - BastionSG
+
+    flavor:
+      ec2: "{{bastion_instance_type}}"
+      azure: "{{bastion_instance_type}}"
+    tags:
+      - key: "AnsibleGroup"
+        value: "bastions"
+      - key: "ostype"
+        value: "linux"
+    rootfs_size: "20"
```

Comment:
`rootfs_size` should be a variable instead of hardcoded value 20

Reply:
Changed to the variable.